src/middleware/visualizer.coffee

```diff
@@ -57,10 +92,20 @@ storeVisualizerEvents = (ctx, done) ->
     ev: 'start'
 
   if ctx.routes
-    addRouteEvents trxEvents, route, 'route' for route in ctx.routes
+    # find TS difference
+    TsDiff = getTsDiff startTS, ctx.routes
```

Comment:
Rather use `tsDiff`. Starting with a capital `T` should rather be used for things like classes.